content/altinn-studio/news/delivered/_index.nb.md

```diff
@@ -36,4 +36,10 @@ hidden: true
         timeline="Q2 2024"
         githubUrl="https://github.com/digdir/roadmap/issues/158"
     >}}
+    {{<news-card 
+        title="Betaling" 
+        content="Mulighet for å konfigurere brukerbetaling for tjenester som er gebyrbelagt eller hvor det er lov å kreve betaling." 
```

Comment:
"Her kan du sette opp brukerbetaling for tjenester som er gebyrbelagt, eller der du ellers har lov til å kreve betaling."